boards/samd21-xpro/Makefile.dep

```diff
@@ -0,0 +1,7 @@
+ifneq (,$(filter gnrc_netdev_default netdev_default,$(USEMODULE)))
+  USEMODULE += at86rf233
+endif
```

Comment:
As far as I see this board does not have radio transceiver, this should be omitted then.